script/unicode/lean-toolchain

```diff
@@ -0,0 +1 @@
+lean4
```

Comment:
This will rely on having a toolchain override called `lean4` set up, right? I guess this is what we need for now, but it would be nice if we can change this to a real toolchain later.

Reply:
What toolchain do you recommend?

Reply:
I think we can just leave out this file. Developers are expected to have their own elan configuration, such as `lean4` in a root `lean-toolchain`.